styles/stylesheet.css

```diff
@@ -0,0 +1,378 @@
+* {
+    margin: 0;
+    padding: 0;
+    font-family: Arial, Helvetica, sans-serif;
+}
+
+h2 {
+    margin-top: 3%;
+}
+
+
+
+h2,
+h3,
+h4,
+.dogs img,
+.dogs p,
+.all-dogs p {
+    margin-bottom: 5%;
+}
+
+.blog1 h2,
+.blog2 h2,
+.blog3 h2,
+.blog1 p,
+.blog2 p,
+.blog3 p {
+    margin-top: 0;
+    margin-bottom: 2%;
+}
+
+/* .blog1 p, .blog2 p , .blog3 p{
+    font-size: .9rem;
+} */
+
+h3,
+h4,
+.dogs p,
+.all-dogs p,
+.dogs button {
+    margin-left: 3%;
+    margin-right: 10%;
+}
+
+header {
+    background-color: black;
+    color: white;
+    background-image: url("../images/header-image.jpg");
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+}
+
+.logo {
+    font-weight: bold;
+    /* margin-left: 2.5%; */
+}
+
+.cart {
+
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-content: center;
+    font-size: .8rem;
+}
+
+header h1 {
+    color: #008A00;
+    font-style: italic;
+    font-size: 1.25rem;
+    /* margin-left: 2.5%; */
+}
+
+.hdr-left {
+    margin-left: 10%;
+    margin-top: 2%;
+    margin-bottom: 5%;
+    /* width:50%; */
+}
+
+.hdr-right {
+    margin-right: 10%;
+    margin-top: 5%;
+    margin-bottom: 5%;
+    /* width:5%; */
+}
+
+header button {
+    padding: 5% 40%;
+    background-color: #008A00;
+    border: 1px solid rgb(48, 44, 44);
+    color: white;
+    font-weight: bold;
+    font-size: .7rem;
+}
+
+header p {
+    font-size: .7rem;
+    padding-top: 30%;
+}
+
+header img,
+header p,
+header button {
+    margin-top: 10%;
+}
+
+.cart img {
+    width: 50%;
+    margin-right: 15%;
+}
+
+nav {
+    background-color: lightgray;
+}
+
+nav ul {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-start;
+    align-content: flex-start;
+    flex-wrap: wrap;
+    padding-left: 10%;
+}
+
+nav li,
+footer li,
+nav p {
+    list-style: none;
+    padding: 1% 0% 1% 1%;
+    font-size: .75rem;
+    font-weight: bold;
+}
+
+nav a,
+footer a {
+    text-decoration: none;
+}
+
+
+#wrapper {
```

Comment:
for CSS the convention is to use classes only. Even if that class is only used once. Change this to a class. 

You'll use id's (and classes) for JS